### ISSUE TITLE
Gradient that goes vertical rather than horizontal?

### ISSUE THREAD

**Issue by MUI68K** [igorkulman/ChangeMenuBarColor#35]
Hi!

Is it possible to have an option that places a gradient on the Y-axis rather than the X-axis? So that it would be possible to make system-wide themes for macOS Big Sur a little for complete with a custom background the best we can do?

I haven't seen documentation for vertical gradients.. please excuse me if you already have that in place.

**Comment by igorkulman** [comment]
Vertical gradients are not possible at the moment, the functionality needs to be added.

**Comment by MUI68K** [comment]
Thanks for writing and answering - I hope it would be something worthwhile trying to implement 😄

**Comment by igorkulman** [comment]
I looked into int and I think it is not possible. If I generate a wallpaper with vertical gradient and set it, macOS just "ignores" the gradient, picks the dominant color from it, applies blur and shows in under the menu bar. 

I verified this behavior by generating the wallpaper, opening it preview to make sure it contains a vertical gradient and everything is correct and then setting it manually.
